hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/utils/LeakDetector.java

```diff
@@ -18,13 +18,15 @@
  */
 package org.apache.hadoop.hdds.utils;
 
+import org.apache.ratis.util.UncheckedAutoCloseable;
```

Comment:
I don't love that Ozone depends on ratis for such a basic need. I guess it's ok for this PR as it's a general issue in Ozone.

Reply:
For now I think of Ratis as a library that provides Raft-implementation as well as useful utilities used by both Ratis and Ozone.  The problem is that we don't have a top-level "utilities" Apache (or Apache-licensed other) project.